example.py

```diff
@@ -519,7 +520,7 @@ def _next_timestamp() -> datetime:
 )
 bundle.append_to_uco_object(extraction_root_directory1)
 
-path_relation_facet1 = uco.observable.FacetPathRelation(path="files/Image/ImG_0123.jpg")
+path_relation_facet1 = uco.observable.FacetPathRelation(path="files/Image/")
```

Comment:
This really should end with the file basename, as a coding-safety mechanism when encountering multiple hard links to a file.

Reply:
@ajnelson-nist Done!

Reply:
I don't think this made it into your update.

Reply:
@ajnelson-nist I corrected the example, and I pushed into the issue-#10 branch. I would delete the branch issue#10 if you agree. Sorry for the mess, I forgot to pull your pushed before covering your comments.